core/store/src/trie/mod.rs

```diff
@@ -891,10 +932,38 @@ impl Trie {
     }
 }
 
+pub fn gen_changes(rng: &mut impl Rng, max_size: usize) -> Vec<(Vec<u8>, Option<Vec<u8>>)> {
```

Comment:
What do you think about moving this and `simplify_changes` into `../test_utils.rs`? And also removing the pub marker from the `tests` mod.

Reply:
++